tests/subsys/nrf_compress/decompression/mcuboot_update/sysbuild/mcuboot/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+
+# Ensure that the qspi driver is disabled by default
+CONFIG_NORDIC_QSPI_NOR=n
```

```suggestion
CONFIG_NORDIC_SPI_NOR=n
```
nRF54l15 hasn't QSPI.